examples/gno.land/r/demo/banktest/banktest.gno

```diff
@@ -40,7 +40,7 @@ func Deposit(returnDenom string, returnAmount int64) string {
 	// return if any.
 	if returnAmount > 0 {
 		banker := std.GetBanker(std.BankerTypeOrigSend)
-		pkgaddr := std.GetOrigPkgAddr()
+		pkgaddr := std.CurrentRealm().Addr()
```

Comment:
why do we modify this testing case?

Reply:
2ae386532fe11158c65b77c9a88eaf9a92a68c70